src/server/webpack.config.js

```diff
@@ -29,7 +29,7 @@ const config = {
   module: {
     loaders: [
       {
-        test: /\.js$/,
+        test: /\.jsx?$/,
```

Comment:
This regex will match both `.js` and `.jsx`.  The `?` makes the `x` optional.

http://regexr.com/3d4vm